### DISCUSSION TITLE
Let's talk about organizing awesome-appwrite!

### DISCUSSION THREAD

**Discussion by NeonSpork** [appwrite/appwrite#2395] [Ideas]
There are _so_ many cool things in the [awesome-appwrite](https://github.com/appwrite/awesome-appwrite) repo! 

However, there is a slight problem now that there are so many good examples, showcases, blogposts, etc. Quite frankly it can be a bit overwhelming to take it _all_ in at once. It would, in my humble opinion, be more useful if the sections were organized into subheadings for frameworks etc. for showcases, and specific appwrite functionality sections for blog posts.

For example in the **Showcase** section (arbitrarily chosen a couple frameworks, organized alphabetically):  

# Showcase
## Angular
- An organized
- List of all
- The cool angular projects
## React
- An organized
- List of all
- The cool react projects
## Svelte
- You get 
- The picture

In the tutorials section, there should also be some sort of general organization into broad categories. Although exactly which categories those should be is something that I believe warrants further discussion. 

Example:  
# Tutorials
- Getting Started for Web (Official)
- Getting Started for Flutter (Official)
- Getting Started for Server (Official)
- Getting Started for Android (Official)
## Announcements
- List of official announcements
- And blog posts
## Authentication and verification
- Cool reading material
## Cloud stuff
- and so on and so forth

Hopefully everyone else agrees that all this wonderful learning material and these awesome showcases should be organized to make them more accessible, and streamline the experience for new (or curious) appwritians!

Excited to hear others' thoughts on the matter!

**Comment by christyjacob4** [comment]
@NeonSpork  This is an amazing idea. We're also considering building a [website](https://madewithreact.com/) out of this to make it more organised. 

What do you think ? Would you be interested in helping with such a feature ?

**Reply by NeonSpork** [reply]
Yes and yes! 

I'd be willing to work on organizing the repo, possibly in collaboration with a core member responsible for the repo?

I'd also be very willing to work on a website or a page for a subdomain/nested page for the main appwrite.io website. I would want to make said website in svelte if possible! :) Working within the existing style and look would be very manageable, and of course I'd defer final sign off on style to a core member responsible for design!

**Reply by NeonSpork** [reply]
Perhaps @PineappleIOnic would be willing to assist in the organization of the items? I feel like a core member should have final say and significant input during the organization 🙂

**Reply by PineappleIOnic** [reply]
I would be happy to assist with organising everything with you! 😄

**Reply by mbos2** [reply]
Should there be a website thing, I'd also like to put some effort into it to help out.

**Comment by eldadfux** [comment]
CC @adityaoberai

**Comment by adityaoberai** [comment]
Hi @NeonSpork 
Absolutely love the idea you have here 😄 
Just want to check once, have you thought of any specific structure for the repo (aside from the example above)?

**Reply by NeonSpork** [reply]
I haven't given it much thought beyond the idea I posted here actually!

It would mostly be a task of organizing into general categories, and then organizing each of those into subcategories if applicable 😃 

I can take a pass at organizing this over the next couple weeks in between another couple projects I'm working on, if you like?

**Reply by NeonSpork** [reply]
I've begun working on this on my own fork, expect a PR and some discussion about how to organize this soon @PineappleIOnic and @adityaoberai  :grin:

**Comment by NeonSpork** [comment]
Alright! I've got a pull review active in the `awesome-appwrite` repo, any specific feedback should probably go [there](https://github.com/appwrite/awesome-appwrite/pull/397)!
